notebooks/examples/CEMS.ipynb

```diff
@@ -0,0 +1,1130 @@
+{
```

Comment:
<p>Let's save the use of <code>Client()</code>  for a bit later as it's more complex and not required. <code>Client()</code>  is only necessary if you want to use multiple cores locally (or a distributed cluster). Dask will serialize the computations on a single core and work just fine without it. You can call <code>my_cems_dd.compute()</code>  instead.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Huh, good to know.</p>